drud/ddev-ui
# Issue 46 build tools and linting

## The Problem/Issue/Bug:
No local development build tools nor code quality tools were part of the project.

## How this PR Solves The Problem:
Webpack with babel and hotreload was implemented, allowing use of ES6 code that's transpiled to vanilla javascript, as well as real time recompiling of assets on save (file watcher). Linting was added as a webpack dependency to ensure code quality and consistency.

## Manual Testing Instructions:

- run make build to generate a new package.json and install node_modules.
- run npm start to load webpack, which should start a node server on localhost and start the ddev-ui
- run npm lint to execute linting and view code issue report

## Related Issue Link(s):
#46
